app/src/org/commcare/dalvik/preferences/CommCarePreferences.java

```diff
@@ -102,38 +104,74 @@
     // Fields for setting print template
     private static final int REQUEST_TEMPLATE = 0;
     public final static String PRINT_DOC_LOCATION = "print_doc_location";
-    private final static String PREF_MANAGER_PRINT_KEY = "print-doc-location";
+
+    private final static String PREFS_APP_SERVER_KEY = "default_app_server";
+    private final static String PREFS_DATA_SERVER_KEY = "ota-restore-url";
+    private final static String PREFS_SUBMISSION_URL_KEY = "PostURL";
+    private final static String PREFS_KEY_SERVER_KEY = "default_key_server";
+    private final static String PREFS_FORM_RECORD_KEY = "form-record-url";
+    private final static String PREFS_UPDATE_FREQUENCY_KEY = "cc-autoup-freq";
+    private final static String PREFS_FUZZY_SEARCH_KEY = "cc-fuzzy-search-enabled";
+    private final static String PREFS_LOCALE_KEY = "cur_locale";
+    private final static String PREFS_PRINT_KEY = "print-doc-location";
```

Comment:
The strings that these keys are set to are used in several places in the code base. Would be nice to have them point to these fields so we only need to change them in one place. not a blocker but would be a good follow up